.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto
```

Comment:
Welcoming suggestions for what else we may want in here, but this should be a good baseline